x/manifest/abci.go

```diff
@@ -0,0 +1,88 @@
+package module
+
+import (
+	"context"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	mintkeeper "github.com/cosmos/cosmos-sdk/x/mint/keeper"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	"github.com/liftedinit/manifest-ledger/x/manifest/keeper"
+	manifesttypes "github.com/liftedinit/manifest-ledger/x/manifest/types"
+)
+
+// BeginBlocker mints new tokens for the previous block.
+func BeginBlocker(ctx context.Context, k keeper.Keeper, mk mintkeeper.Keeper, bk bankkeeper.Keeper) error {
+
+	ic := minttypes.DefaultInflationCalculationFn
+
+	defer telemetry.ModuleMeasureSince(manifesttypes.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
+
+	// fetch stored minter & params
+	minter, err := mk.Minter.Get(ctx)
+	if err != nil {
+		return err
+	}
+
+	params, err := mk.Params.Get(ctx)
+	if err != nil {
+		return err
+	}
+
+	// recalculate inflation rate
+	totalSupply := bk.GetSupply(ctx, "umfx").Amount
+
+	// bondedRatio, err := k.BondedRatio(ctx)
+	// if err != nil {
+	// 	return err
+	// }
+	// always 0 for us
+	bondedRatio := sdkmath.LegacyZeroDec()
```

Comment:
With poa v0.0.1-alpha.4 tag, this can be re-used. or should be set to 1.0? or forced inflation? not sure yet